config/routes.rb

```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  get 'static_pages/home'
+  get 'static_pages/about'
+  get 'static_pages/contact'
```

Comment:
These 3 lines should be combined into one namespace. Something like:

```ruby
namespace :static_pages do
  get '/home'
  get '/about'
  get '/contact'
end
```

Reply:
👍